history/history-network.md

```diff
@@ -261,17 +259,27 @@ content_key         = selector + SSZ.serialize(receipt_key)
 
 Note: The type-specific receipts encoding might be different for future receipt types, but this content encoding is agnostic to the underlying receipt encodings.
 
-#### Epoch Record
+#### Block Header Indexes
```

Comment:
I would call it also here Block Header by Number (and perhaps refer to the other one as by hash)